cmake/macros/SetPackagingParameters.cmake

```diff
@@ -49,8 +49,8 @@ macro(SET_PACKAGING_PARAMETERS)
     set(PRODUCTION_BUILD 1)
     set(BUILD_VERSION ${RELEASE_NUMBER})
     set(BUILD_ORGANIZATION "Vircadia")
-    set(HIGH_FIDELITY_PROTOCOL "hifi")
-    set(HIGH_FIDELITY_APP_PROTOCOL "hifiapp")
+    set(HIGH_FIDELITY_PROTOCOL "vw")
```

Comment:
I am heavily against calling it `vw://` as it will make is impossible for a user to ever find information about it on any big search engine. Problem is that someone that doesn't know Vircadia won't be able to find out what a `vw://` link is for. Considering how little people use links like this nowadays, people that use Vircadia will most likely also not know what that is for. I am pretty sure that even on a `hifi://` link, most fairly new users will go "What is this?".

Personally I am for `vircadia://` as that leaves no room for interpretation and will yield good results on all major search engines.